src/util/BaseLoginController.js

```diff
@@ -100,7 +100,10 @@ define(['okta', 'q'], function (Okta, Q) {
 
     postRenderAnimation: function () {
       // Event triggered after a page is rendered along with the classname to identify the page
+      // TODO: Deprecate this event in the next major version in favor of 'render'
       this.trigger('pageRendered', {page: this.className});
+
+      this.trigger('render', { controller: this.className });
```

Comment:
Sorry to come back to naming, but I think this should be `rendered` and not `render`. As a developer, I would expect that an event named `render` is triggered when the `render` function is called, not in the `postRender`.